index.js

```diff
@@ -14,4 +13,29 @@ app.get('/', (req, res) => res.send("Hello"));
 function onListen() {
     console.log(`Listening on port ${port}`);
 }
-app.listen(port, onListen);
\ No newline at end of file
+app.listen(port, onListen);
+
+// connect to heroku database
+const { Client } = require('pg');
+
+const client = new Client({
+    host: 'ec2-18-233-32-61.compute-1.amazonaws.com',
+    port: 5432,
+    user: 'gadyrjddeeskwc',
+    password: '8a6d028082f3a13122125822711519bb0b547f597301b1e6006f89978c04e5aa',
+    database: 'd4i5njekim7upa',
+    ssl: {
+        rejectUnauthorized: false
+      }
+  })
+
+client.connect();
+
+// print test_table
+client.query('SELECT * FROM test_table;', (err, res) => {
+  if (err) throw err;
+  for (let row of res.rows) {
+    console.log(JSON.stringify(row));
```

Comment:
Lmao 
![image](https://user-images.githubusercontent.com/25494065/82981312-94e76d80-9fa0-11ea-8a8e-6dde2f4053bd.png)

Interesting the `console.log` goes to the Heroku logs and not the browser